app/views/layouts/course.html.slim

```diff
@@ -2,12 +2,10 @@
 = render within_layout: layout do
   = render_breadcrumbs
   div.row
-    div.col-lg-2.col-md-3.col-sm-4
-      nav role='navigation'
-        div.collapse.navbar-collapse#course-navigation-navbar
-          = sidebar_items(controller.sidebar_items(type: :normal))
-          h5 = t('.administration')
-          = sidebar_items(controller.sidebar_items(type: :admin))
+    = sidebar do
+      = sidebar_items(controller.sidebar_items(type: :normal))
+      h5 = t('.administration')
```

Comment:
I see it as semantic -- HTML should be semantic, styling is a layer over that. Consider this case: what if there are multiple sidebars on a page?